docs/src/tutorials/mathprogbase.md

```diff
@@ -0,0 +1,96 @@
+# Transitioning from MathProgBase
+
+MathOptInterface is a replacement for [MathProgBase.jl](https://github.com/JuliaOpt/MathProgBase.jl).
+However, it is not a direct replacement.
```

Comment:
Isn't https://github.com/jump-dev/MatrixOptInterface.jl providing a closer interface to what MPB looked like?

Reply:
For the modeling side, yes. But it still needs some development to integrate properly, so I held off mentioning it.